rubicon_ml/client/project.py

```diff
@@ -228,26 +230,42 @@ def log_experiment(
 
         return Experiment(experiment, self)
 
-    def experiment(self, id):
-        """Get an experiment logged to this project by id.
+    def experiment(self, name=None, id=None):
```

Comment:
I know I mentioned that the order here wouldn't matter, but lets actually switch it so `id` is first for the sake of backwards compatibility. that way users that upgrade won't explicitly have to add the `id` kwarg name to places where they currently have `project.experiments("some-id-string")` 

its still good to explicitly use the `id` kwarg name in the places you changed in the library already just to be explicit, so no need to change those back